manifests/r/rickbutton/workspacer/beta/0.9.12/rickbutton.workspacer.beta.installer.yaml

```
@@ -16,7 +16,7 @@ UpgradeBehavior: install
 Installers:
 - Architecture: x64
   InstallerUrl: https://github.com/workspacer/workspacer/releases/download/unstable/workspacer-latest-unstable.msi
-  InstallerSha256: EFBA59BDD7BFA7CF7ADBF06E624E3C76960029B18EEEA1354FE1CBC4DB963F8F
-  ProductCode: '{84154A45-9EDD-499B-BC7D-8780B8CC4182}'
+  InstallerSha256: D8B0A56821BEE5099787D02DCF1D51999266755F4E152A6E737F45656CFE9CC0
+  ProductCode: '{B2C001A5-4F45-48A7-A05F-21DF946FF07E}'

```

The Product Code has changed to `6CC96BF5-8FE6-4E58-BFAA-5C7CBC0405D8`.

```suggestion
  ProductCode: '{6CC96BF5-8FE6-4E58-BFAA-5C7CBC0405D8}'
```